tests/test_watcher.py

```
@@ -24,11 +24,17 @@ def watcher_instance():
+        "AutoAccept": {
+            "enabled": False,
+            "job_sources": "rss,websocket",
+            "min_reward": 0.0,
+            "max_reward": 999999.0,
+        },
     }
-    mock_config.get.side_effect = (
-        lambda section, key, **kwargs: config_data.get(section, {}).get(key)
-    )
-    mock_config.config = config_data  
+    mock_config.get.side_effect = lambda section, key, **kwargs: config_data.get(
```

Ruff (0.14.13)

Unused lambda argument: `kwargs`

(ARG005)